.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+
+### Description
+Please provide a clear and concise description of your changes. Explain **what** this pull request accomplishes and *
+*why** it is needed.
+
+---
+
+### Related Issue
+
+If this pull request addresses an existing issue, please link to it below. Use the `Fixes #<issue-number>` syntax to
```

Comment:
Thanks for your help, already good - a few suggestions to make it even better.
Fixes could point as a link to https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue for those not aware of the details.